core/cppgc.rs

```diff
@@ -58,6 +132,78 @@ pub fn make_cppgc_object<'a, T: GarbageCollected + 'static>(
   wrap_object(scope, obj, t)
 }
 
+pub fn make_cppgc_object2<
```

Comment:
Maybe change a name to something more descriptive to suggest that it uses the second argument as a prototype

Reply:
It's named because its easier for the macro to do: `format!("make_cppgc_object{}", chain.len())`. These aren't supposed to be publicly used. I've added `#[doc(hidden)]`